### ISSUE TITLE
Images in the gallery don't have alt text

### ISSUE THREAD

**Issue by merapi** [GoogleForCreators/web-stories-wp#1092]
## Bug Description

```
Expected page to pass Axe accessibility tests.
    Violations found:
    Rule: "image-alt" (Images must have alternate text)
    Help: https://dequeuniversity.com/rules/axe/3.5/image-alt?application=axe-puppeteer
    Affected Nodes:
      .mediaElement__Image-sc-1a0ilt5-0
        Fix ANY of the following:
        - Element does not have an alt attribute
        - aria-label attribute does not exist or is empty
        - aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
        - Element has no title attribute or the title attribute is empty
        - Element's default semantics were not overridden with role="presentation"
        - Element's default semantics were not overridden with role="none"
```

## Expected Behaviour

The image should have an alt text.

## Steps to Reproduce

Run `npm run test:e2e` with some images on your local env.

## Screenshots

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by spacedmonkey** [comment]
There is alt text provided by WP / API. We should just reuse that.

**Comment by merapi** [comment]
> There is alt text provided by WP / API. We should just reuse that.

If you upload an image right now, `alt_text` is empty, you just have `title`(filename without ext.) - should we use it (`alt_text || title`)? (it's not a decorative image, so `alt` shouldn't be empty).

https://www.w3.org/TR/WCAG20-TECHS/H67.html

**Comment by spacedmonkey** [comment]
Using the alt from the API should be a good start.
